scripts/drone/steps/lib.star

```diff
@@ -775,11 +775,34 @@ def lint_frontend_step():
         "commands": [
             "yarn run prettier:check",
             "yarn run lint",
-            "yarn run i18n:compile",  # TODO: right place for this?
             "yarn run typecheck",
         ],
     }
 
+def verify_i18n_step():
+    extract_error_message = "Extraction failed. Make sure that you have no dynamic translation phrases, such as 't(\\`preferences.theme.\\$\\${themeID}\\`, themeName)' and that no translation key is used twice."
```

Comment:
This is making me confused.
1. Escape character for Starlark
2. Escape character for Drone
3. Escape character for Bash